Library/CMakeLists.txt

```diff
@@ -27,8 +27,10 @@ set(ENABLE_SPVREMAPPER OFF CACHE BOOL "Enables building of SPVRemapper")
 set(ENABLE_AMD_EXTENSIONS OFF CACHE BOOL "Enables support of AMD-specific extensions")
 set(ENABLE_GLSLANG_BINARIES OFF CACHE BOOL "Builds glslangValidator and spirv-remap")
 set(ENABLE_NV_EXTENSIONS OFF CACHE BOOL "Enables support of Nvidia-specific extensions")
-set(ENABLE_HLSL OFF CACHE BOOL "Enables HLSL input support")
 set(ENABLE_OPT OFF CACHE BOOL "Enables spirv-opt capability if present")
+if(WIN32 OR WINDOWS_STORE)
+    set(ENABLE_HLSL ON CACHE BOOL "Enables HLSL input support")
+endif()
```

Comment:
This should not be here. ENABLE_HLSL should be set to OFF always as it is for input and our input is always GLSL.

Reply:
Also, some of these settings have been removed and are no longer necessary.